openedx_events/learning/signals.py

```diff
@@ -196,3 +197,58 @@
         "notification_data": UserNotificationData,
     }
 )
+
+# .. event_type: org.openedx.learning.exam.attempt.submitted.v1
+# .. event_name: EXAM_ATTEMPT_SUBMITTED
+# .. event_description: Emitted when an exam attempt is submitted by a learner.
+# .. event_data: ExamAttemptData
+EXAM_ATTEMPT_SUBMITTED = OpenEdxPublicSignal(
+    event_type="org.openedx.learning.exam.attempt.submitted.v1",
+    data={
+        "exam_attempt_data": ExamAttemptData,
+    }
+)
+
+# .. event_type: org.openedx.learning.exam.attempt.rejected.v1
+# .. event_name: EXAM_ATTEMPT_REJECTED
+# .. event_description: Emitted when an exam attempt is marked rejected by an instructor or staff.
+# .. event_data: ExamAttemptData
+EXAM_ATTEMPT_REJECTED = OpenEdxPublicSignal(
+    event_type="org.openedx.learning.exam.attempt.rejected.v1",
+    data={
+        "exam_attempt_data": ExamAttemptData,
+    }
+)
+
+# .. event_type: org.openedx.learning.exam.attempt.verified.v1
+# .. event_name: EXAM_ATTEMPT_VERIFIED
+# .. event_description: Emitted when an exam attempt is marked verified by an instructor or staff.
```

Comment:
same as the above comment. Right now an LTI tool can automatically trigger this if there is no suspicious behavior.